src/includes/control.h

```diff
@@ -249,8 +251,9 @@ inline dasynq::rearm control_conn_cb(eventloop_t * loop, control_conn_watcher *
         }
     }
     
+    constexpr int outbuf_limit = 16384; // 16KB
```

Comment:
> Can you please also use a named constant (a constexpr int at the top of the file is fine) 

Please place near the top of the file as I asked originally and use a name in all caps so it is clear that it is a constant (just like `IN_EVENTS` etc).

Reply:
(The point of making it a named constant is so that it is easy to find and modify and it can be used elsewhere in the code if needed. If you put it right next to the code that uses it these benefits are lost).